### PR TITLE
Ft/refactor image class

### PR DESCRIPTION
**Pull request by danielerat**
@Moussa-Kalam @izzett222 @gitarecy @CynthiaUmuhire 
# Refactoring all the image alt names and removing the unused/empty classes. 
### Checklist before requesting a review
- [x] Renaming the alt attributes from empty name to a meaningful one.
- [x] Removing all the unused classes from the codebase (class="")

### CODE REVIEW

**Review by CynthiaUmuhire — APPROVED** [review]
The images are more descriptive

File: src/sections/rwanda.html
Comment (gitarecy) [inline src/sections/rwanda.html:1]: rename the vectors better, something like 'team-highlighter"'

**Review by gitarecy — CHANGES_REQUESTED** [review]

File: src/sections/managed_teams.html [hunk 1]
```diff
@@ -174,7 +174,11 @@ <h2 class="pb-3 lg:text-[2rem] lg:leading-[2.5rem] w-[100%]">
         </div>
       </div>
       <div class="basis-1/4 lg:basis-1/6 md:1/4">
-        <img src="src/Assets/logos/maibornwolff.svg" class="w-full" alt="" />
+        <img
+          src="src/Assets/logos/maibornwolff.svg"
+          class="w-full"
+          alt="Mailborn wolff icon"
```
Comment (Moussa-Kalam) [inline src/sections/managed_teams.html]: Maibornwolff instead of maiLborn

File: src/sections/partners.html [hunk 2]
```diff
@@ -16,18 +16,26 @@ <h3 class="font-bold md:text-[2.25rem]">
       class="grid gap-x-36 gap-y-16 place-items-center m-auto font-montserrat sm:grid-cols-2 w-full lg:w-[70%] mt-20"
     >
       <div id="maibornwolff">
-        <img src="src/Assets/logos/maibornwolff.svg" alt="" />
+        <img src="src/Assets/logos/maibornwolff.svg" alt="Mailbornwolff logo" />
```
Comment (Moussa-Kalam) [inline src/sections/partners.html]: Maibornwolff. 
Please remove the "L" before the letter "b"

File: src/sections/partners.html [hunk 3]
```diff
@@ -16,18 +16,26 @@ <h3 class="font-bold md:text-[2.25rem]">
...
       </div>
       <div id="stihl" class="grayscale">
         <div class="flex gap-4">
           <div>
-            <img src="src/Assets/logos/bmw.svg" alt="" class="w-full" />
+            <img src="src/Assets/logos/bmw.svg" alt="BMW log" class="w-full" />
```
Comment (Moussa-Kalam) [inline src/sections/partners.html]: logo

File: src/sections/partners.html [hunk 4]
```diff
@@ -39,21 +47,21 @@ <h3 class="font-bold md:text-[2.25rem]">
 
       <!-- --------------------------------------------- -->
       <div id="maliyo">
-        <img src="src/Assets/logos/maliyo-games.svg" alt="" />
+        <img src="src/Assets/logos/maliyo-games.svg" alt="Maliyo games" />
```
Comment (Moussa-Kalam) [inline src/sections/partners.html]: Maliyi games logo

File: src/sections/rwanda.html
Comment (danielerat) [reply src/sections/rwanda.html:1]: Yes @gitarecy On it.

File: src/sections/rwanda.html [hunk 5]
```diff
@@ -65,23 +65,35 @@ <h1 class="font-extrabold text-sm mb-7">
...
         <div class="basis-4/6">
           <h3 class="font-bold">stetig wachsender Markt und Talentpool</h3>
         </div>
       </div>
       <div class="flex flex-col lg:flex-row items-center gap-3">
         <div class="basis-6/6 shrink-0 mx-auto">
-          <img src="src/Assets/icons/Vector-2.svg" alt="" class="w-full" />
+          <img
+            src="src/Assets/icons/Vector-2.svg"
+            alt="Vector Image"
```
Comment (Moussa-Kalam) [inline src/sections/rwanda.html]: Could you please make it more descriptive? Thanks

File: src/sections/rwanda.html [hunk 6]
```diff
@@ -65,23 +65,35 @@ <h1 class="font-extrabold text-sm mb-7">
...
         <div class="basis-4/6">
           <h3 class="font-bold text-gray-400">sehr gutes Investitionsklima</h3>
         </div>
       </div>
       <div class="flex flex-col lg:flex-row items-center gap-3">
         <div class="basis-6/6 shrink-0 mx-auto">
-          <img src="src/Assets/icons/Vector-3.svg" alt="" class="w-full" />
+          <img
+            src="src/Assets/icons/Vector-3.svg"
+            alt="Vector line image"
```
Comment (Moussa-Kalam) [inline src/sections/rwanda.html]: Please give it a more descriptive alt message,

**Review by Moussa-Kalam — CHANGES_REQUESTED** [review]
Great work overall. 
Please add the request changes. 
Kudos ✨.

**Review by gitarecy — APPROVED** [review]
All looks good and cleaner

**Review by Moussa-Kalam — APPROVED** [review]
Good!

**Pull request merged**